osf/models/notification_type.py

```diff
@@ -115,30 +108,23 @@ class Type(str, Enum):
         ADDON_FILE_COPIED = 'addon_file_copied'
         ADDON_FILE_RENAMED = 'addon_file_renamed'
         ADDON_FILE_MOVED = 'addon_file_moved'
-        ADDON_FILE_REMOVED = 'addon_file_removed'  # unused
+        ADDON_FILE_REMOVED = 'addon_file_removed'
         FOLDER_CREATED = 'folder_created'
 
         # Provider notifications
         PROVIDER_NEW_PENDING_SUBMISSIONS = 'provider_new_pending_submissions'
         PROVIDER_NEW_PENDING_WITHDRAW_REQUESTS = 'provider_new_pending_withdraw_requests'
         PROVIDER_REVIEWS_SUBMISSION_CONFIRMATION = 'provider_reviews_submission_confirmation'
-        PROVIDER_REVIEWS_MODERATOR_SUBMISSION_CONFIRMATION = 'provider_reviews_moderator_submission_confirmation'  # unused
-        PROVIDER_REVIEWS_REJECT_CONFIRMATION = 'provider_reviews_reject_confirmation'  # unused
-        PROVIDER_REVIEWS_ACCEPT_CONFIRMATION = 'provider_reviews_accept_confirmation'  # unused
         PROVIDER_REVIEWS_RESUBMISSION_CONFIRMATION = 'provider_reviews_resubmission_confirmation'
-        PROVIDER_REVIEWS_COMMENT_EDITED = 'provider_reviews_comment_edited'  # unused
-        PROVIDER_CONTRIBUTOR_ADDED_PREPRINT = 'provider_contributor_added_preprint'  # unused
         PROVIDER_CONFIRM_EMAIL_MODERATION = 'provider_confirm_email_moderation'
         PROVIDER_MODERATOR_ADDED = 'provider_moderator_added'
-        PROVIDER_CONFIRM_EMAIL_PREPRINTS = 'provider_confirm_email_preprints'  # unused
```

Comment:
Let's confirm the replacement for this one. As discussed, we have `user_confirm_email_preprints` in the `yaml` file but not in the types definition. Please double check.

We may have other ones in a similar case, please take a look.